### DISCUSSION TITLE
Volunteer testing discussion

### DISCUSSION THREAD

**Discussion by jgh9** [Versent/saml2aws#1038] [General]
Let's go over pre-release validation per #1029 
@gliptak 
@mapkon 

Current [expectations](https://github.com/Versent/saml2aws/pull/1029#issuecomment-1533977671) linked and noted below:

> Code works as expected
> Existing features are not broken
> New features integrate properly

**Comment by mikex99** [comment]
How do I get a copy of the test binary?

**Comment by gliptak** [comment]
Scroll down to artifacts at https://github.com/Versent/saml2aws/actions/runs/4857817432

The ZIP is expected to contain the variants

**Comment by chris-reeves** [comment]
I've done manual testing for my use case (Shibboleth provider) on my platform (Linux/amd64).

The test suite has run successfully for the various build platforms (ubuntu-latest, macOS-latest, macos-11) as per the actions run that @gliptak refers to.

As far as I can tell, other than dependabot version bumps and test suite improvements the main changes are my build changes (#1029, which basically revert the default options/flags to what they were before v2.36.6) and #1006 (which also appears to be related to reverting to previous functionality, this time for downloading browser drivers for Playwright).

I'm not sure there much further testing that I can do - I've exercised all the functionality that I use.

**Comment by jgh9** [comment]
I have verified manual testing per my use case (Shibboleth provider) on platform Amazon Linux 2. I am not aware of other features to test, but have verified via standard operations. If there are any new features I can help verify, please note how these can be tested and I can attempt to confirm.

**Comment by mikex99** [comment]
I've tested on Windows and Mac using the Browser method and everything
worked as expected when using the download flag.  I haven't tried inserting
it into the config file.

**Comment by mapkon** [comment]
This has been such a tremendous response. I will cut the v2.36.7 release
